ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -224,9 +230,7 @@ describe('EthOverview', () => {
 
       await waitFor(() =>
         expect(openTabSpy).toHaveBeenCalledWith({
-          url: expect.stringContaining(
-            'https://metamask-institutional.io/swap',
-          ),
+          url: 'https://metamask-institutional.io/swap',
```

Comment:
Why not this? `${mmiPortfolioUrl}/swap`

Reply:
Its in a test, we want to see the value output.

Reply:
I know, but if the value changes, the test will fail, is it what we want?